depends/patches/qt/.configure

```diff
@@ -0,0 +1,17 @@
+#! /bin/sh
```

Comment:
This files looks to be stripped of its licence (LGPL). It's better to extract the file (and others) directly from the submodules. If the files have been slightly modified from the Qt source add a patchfile - to be in line with the current practice.

Reply:
These specific files are not within the submodules unfortunately nor are they listed in the download directory see :- https://download.qt.io/official_releases/qt/5.9/5.9.8/submodules. They are only in the all-in-one or the git, so I had copied them separately to avoid downloading the whole all-in-one only to extract these three files and then duplicate the work by separately downloading each module.

Reply:
In that case I vote for downloading the all-in-one. It would reduce this PR by three files and most of its lines. It would also simplify existing code. The download would take a bit longer but if we want QML I think it's worth it.

Reply:
I'm already working on that approach @icota. Also worth noting that single file was the old approach, and it was changed in ab67dd7818ff2d0910f3fd9bfca9412d85de4424, maybe @theuni had a motivation for using modules.

Reply:
@promag I have an all in one build already functional, should i open a PR and we collaborate ?

Reply:
@BlockMechanic sure! Happy to review and compare with my change.